postr/youtube_postr.py

```diff
@@ -54,36 +54,84 @@ def get_refresh_token(self) -> Any:
     def post_text(self, text: str) -> bool:
         ''' This method takes in the text the user want to post
         and returns the success of this action'''
+        # No text to be posted on YouTube
         return True
 
     def post_video(self, url: str, text: str) -> bool:
         ''' This method takes in the url for the video the user
         want to post and returns the success of this action'''
+        self.upload_video(url, text, text, 22, '', 'public',)
         return True
 
+    def upload_video(
+        self,
+        file: str, title: str, description: str,
+        category: int, keywords: str, privacy_status: str,
+    ) -> Any:
+        parser = argparse.ArgumentParser()
```

Comment:
why are you using argparser inside a method in a non-command line application?

Reply:
Good point, that is a holdover from the command line parser video uploading sample code I pulled. I have fixed it in the new commit.